File: src/main/kotlin/utils/Env.kt
package utils

import io.github.cdimascio.dotenv.Dotenv

object Env {
    private val dotEnv = Dotenv.load()
    
    val openAiApiKey = dotEnv["OPENAI_API_KEY"]!!
    val gitHubApiKey = dotEnv["GITHUB_API_KEY"]!!
}

File: backend/lib/core/utils/env.dart
import 'package:dotenv/dotenv.dart';

class Env {
  static final openAiApiKey = _dotEnv['OPENAI_API_KEY']!;
  static final gitHubApiKey = _dotEnv['GITHUB_API_KEY']!;
}

final _dotEnv = DotEnv()..load();
